api/pom.xml

```diff
@@ -3,7 +3,7 @@
 	<modelVersion>4.0.0</modelVersion>
     <parent>
 		<groupId>org.openmrs.module</groupId>
-		<artifactId>outgoing-message-exceptions</artifactId>
```

Comment:
Please, don't change module id. There is documentation how to add fragment controller with custom configuration.